iceoryx_posh/test/moduletests/test_mq_interface.cpp

```diff
@@ -381,4 +411,10 @@ TEST_F(CMqInterface_test, MqInterfaceCreator_IsInitialized)
     CMqInterface_Open(base);
     CMqInterface_IsInitialized<MqInterfaceCreator>(base);
 }
+
+TEST_F(CMqInterface_test, DestroyMessagequeueReturnsIsInitialisedFalse)
+{
+    MqInterfaceCreator base(ifName);
+    CMqInterface_cleanupResource(base);
+}
 #endif
```

Comment:
Please consider what the tests are intended to check and write them accordingly. The extra indirection of the inner calls appear to be redundant and misleading.

Reply:
I think in this case all the test file should be refactoring not only the added test